tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -130,6 +133,42 @@ static void InitializeConnectionMethods(py::class_<DuckDBPyConnection, shared_pt
 	         "Run a SQL query. If it is a SELECT statement, create a relation object from the given SQL query, "
 	         "otherwise run the query as-is.",
 	         py::arg("query"), py::arg("alias") = "query_relation")
+	    .def("read_csv", &DuckDBPyConnection::ReadCSV, R"END_OF_COMMENT(
+                Read a CSV file
+                
+                :param name:
+                    The name used to identify the CSV file
+                :type name: ``str``
+                :param \**kwargs:
+                    See below
+                
+                :Keyword Arguments:
+                    * *header* (``bool``) --
```

Comment:
Is there a reason you haven't formatted these using the Sphinx syntax, as you've done with the arguments above?

Reply:
https://stackoverflow.com/a/27724915

I'll try to write it using None though